lib/fluxsink/scpfluxsink.cc

```diff
@@ -138,6 +137,7 @@ class ScpFluxSink : public FluxSink
 			}
 		}
 
+		_fileheader.revolutions = revolution - 1;
```

Comment:
I think this is off by 1

Reply:
Yes, you're right.

Reply:
It's worse than that, there's no `break` inside the loop, so it always writes five revolutions...